lib/client/clock-client.js

```diff
@@ -44,7 +44,7 @@ client.render = function render (xhr) {
       rec = element;
     }
     else if (element.sgv && rec && !delta) {
-      delta = (rec.sgv - element.sgv)/((rec.date - element.date)/(5*60*1000));
+      delta = ((rec.sgv - element.sgv)/((rec.date - element.date)/(5*60*1000))).toFixed(0);
```

Comment:
toFixed returns a string. This should be cast back to a number, or you should use some other method to round the digits to produce a number

Reply:
Oops saved half-written comment - meant to say we should fix the !delta comparison and not rely on the number being cast to a string, as this is inherently fragile.

Reply:
As a DB developer I'm allergic to null comparisons, but it should work.
Also removed the !delta condition from the previous if statement as it's not needed there.